eland/dataframe.py

```diff
@@ -1621,6 +1621,57 @@ def groupby(
             by=by, query_compiler=self._query_compiler.copy(), dropna=dropna
         )
 
+    def quantile(
+        self,
+        q: Union[float, List[float]] = 0.5,
+        numeric_only: Optional[bool] = True,
+    ) -> "pd.DataFrame":
+        """
+        Used to calculate quantile for a given DataFrame.
+
+        PARAMETERS
+        ----------
+        q:
+            float or array like, default 0.5
+            Value between 0 <= q <= 1, the quantile(s) to compute.
+        numeric_only: {True, False, None} Default is None
+            Which datatype to be returned
+            - True: Returns all values as float64, NaN/NaT values are removed
+            - None: Returns all values as the same dtype where possible, NaN/NaT are removed
+            - False: Returns all values as the same dtype where possible, NaN/NaT are preserved
+
+        Returns
+        -------
+        pandas.DataFrame
+            quantile value for each column
+
+        See Also
+        --------
+        :pandas_api_docs:`pandas.DataFrame.quantile`
+
+        Examples
+        --------
+        >>> ed_flights = ed.DataFrame('localhost', 'flights')
+        >>> ed_flights.quantile() # doctest: +SKIP
```

Comment:
Is this measurement not stable? if it is stable we can remove the `+SKIP`

Reply:
It's not stable, Somehow i forgot to add skip for one doctest and It failed. check the builds. :)